scripts/mkimage

```diff
@@ -255,6 +255,15 @@ elif [ "$BOOTLOADER" = "u-boot" ]; then
     fi
 fi # bootloader
 
+# run device or project specific mkimage
+  if [ -n "$DEVICE" -a -f $PROJECT_DIR/$PROJECT/devices/$DEVICE/mkimage ]; then
+    echo "image: running device mkimage..."
+    . $PROJECT_DIR/$PROJECT/devices/$DEVICE/mkimage
+  elif [ -f $PROJECT_DIR/$PROJECT/mkimage ]; then
+    echo "image: running project mkimage..."
+    . $PROJECT_DIR/$PROJECT/mkimage
```

Comment:
What/where is this `mkimage` script?

I think we should put scripts in a `scripts` directory, rather than drop them in the root of a top level-ish directory.
ie.
`$PROJECT_DIR/$PROJECT/devices/$DEVICE/scripts/mkimage`
`$PROJECT_DIR/$PROJECT/scripts/mkimage`